app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -33,8 +35,13 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        DaggerActivityComponent.builder()
```

Comment:
I'm confused by this. Why can't the application encapsulate the primary object graph? 

Also, what is the benefit of this secondary object graph here? We're not even holding onto it after injection.